src/x11/screen.c

```diff
@@ -37,31 +37,42 @@ static int x_follow_tear_down_error_handler(void);
 static int FollowXErrorHandler(Display *display, XErrorEvent *e);
 static Window get_focused_window(void);
 
+
+/**
+ * A cache variable to cache the Xft.dpi xrdb values.
+ * We do not expect to change xrdb often, but there's much
+ * overhead to query it once.
+ *
+ * @retval DBL_MIN: uncached
+ * @retval     <=0: Invalid and unusable value
+ * @retval      >0: valid
+ */
+double screen_dpi_xft_cache = DBL_MIN;
```

Comment:
Sorry for the misleading review earlier `DBL_MIN` is the smallest _positive_ value, should've double checked that one :/